engine/variableValue.h

```diff
@@ -139,19 +141,28 @@ namespace minsky
       setXVector(std::move(xv));
       return d;
     }
-    size_t numElements() const {
+    // For feature 47
+    std::vector<size_t> index;
+    
+    double sparsityRatio=0.0;       
```

Comment:
Do we need this data member?

Reply:
No, we don't. I can just have a local variable in CSVParser.cc to decide whether or not the data is sparse. I will remove it.